source/_components/cover.garadget.markdown

```diff
@@ -89,3 +89,12 @@ customize:
     icon: mdi:wifi
 ```
 
+Some of the Garadget sensors can create a lot of clutter in the logbook.  Use this section of code in your `configuration.yaml` to exclude those pesky entries.
```

Comment:
I would prefer if you remove the word pesky to make this look a bit more professional.